docs/application/dotnet/guides/nui/grid-layout.md

```diff
@@ -1,4 +1,112 @@
-# Grid Layout
+# Grid layout(Tizen 6.0 or later)
+
+> [!NOTE]
+> If you are using GridLayout on Tizen 5.5, Please refer to [Grid Layout(Tizen 5.5)](#gridlayout-tizen5.5) instead.
+
+`GridLayout` is a grid box for the two dimensional layout. It constraints the x and y position, width, and height of the child actors.
```

Comment:
> **Also is this line needed in the intro section, the property is introduced after this sentence so I feel this is not needed here yet**
> If the `Column` or the `Row` Property is not specified on the child, it is automatically assigned left to right or top to bottom depending on the `GridOrientation` Property.

Since the same description is in `Columns and rows` section, i think we can remove this line here.